build.gradle

```diff
@@ -35,18 +37,23 @@ test {
 
 group = "org.jenkins-ci.plugins"
 description = "Allows the dynamic launch Jenkins agent on a Mesos cluster, depending on workload"
-sourceCompatibility = 1.11
-targetCompatibility = 1.11
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 jenkinsPlugin {
-    coreVersion = "2.155"
+    coreVersion = "2.173"
```

Comment:
can we set to 2.164? that maps to the next jenkins version we have queued up for release to the universe. Jenkins doesn't garuntee downgrade

https://github.com/mesosphere/dcos-jenkins-service/blob/3.5.x/Dockerfile#L1

Reply:
I'll try.